Dialogue/Afin/np_010030.csv

```diff
@@ -19,70 +19,70 @@ Afin_favs_present_080#0,"""Received \""Homing Emission\"" from Afin."""
+Afin_19_020#0,"""...I feel like there is something wrong with you if you think that that is normal."""
 11_voice_NPC_5003_104#5,"""I didn't join ARKS because I wanted to protect someone."""
-Afin_20_020#0,"""Like I thought, if we get attacked here I end up thinking I want to protect everyone. It's weird to think about."""
-11_voice_NPC_03_104#0,"""Ever since Dark Falz has been revived, everyone's been on edge."""
-Afin_mb9_020#0,"""Well, I might be stating the obvious but it's no use feeling down about it."""
+Afin_20_020#0,"""Like, I thought if we get attacked here, I end up thinking I want to protect everyone. It's weird to think about."""
+11_voice_NPC_03_104#0,"""Ever since Dark Falz's revival, everyone's been on edge."""
+Afin_mb9_020#0,"""Well, I might be stating the obvious, but it's no use feeling down about it."""
 11_voice_NPC_03_106#0,"""Seems like there are a lot of rumours flying around."""
-Afin_mb10_020#0,"""There are some super strong people out there so be careful."""
+Afin_mb10_020#0,"""There are some super-strong people out there, so be careful."""
 11_voice_NPC_03_104#1,"""Well, I think it's good to let off a little steam in the ARKS Battle Tournament every once and a while."""
 Afin_mb11_020#0,"""Looks like a lot of people are joining in...what should I do?"""
 11_voice_NPC_5003_105#3,"""...Ah, my bad partner. I was spacing out."""
 Afin_mb12_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
 11_voice_NPC_5003_105#4,"""...Ah, my bad partner. I was spacing out."""
 Afin_mb13_020#0,"""I know I can't solve anything just by thinking about it...but I'm bad at acting on my thoughts."""
 11_voice_NPC_5003_104#6,"""The Quarry on Planet Lillipa....I feel like there's something there."""
-Afin_mb14_020#0,"""Ah, it's not really a 'get rich quick' thing. That gets old pretty fast. """
-11_voice_NPC_5003_104#7,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-Afin_mb15_020#0,"""If that's the case then I should just stop suffering....."""
-Afin_mb15_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-Afin_mb16_010#0,"""This...seems like a joke. My sister that I was separated from is Dark Falz...what's happening..."""
-11_voice_NPC_5003_105#5,"""If that's the case then I should just stop suffering....."""
-Afin_mb16_030#0,"""....no, if you're going to keep living then things will work out. I'm sure, everything will be fine."""
-11_voice_NPC_5003_104#8,"""...partner. That turned out to be amazing."""
-Afin_mb16zeno_020#0,"""You really seem to get caught up in everything. ...I feel sorry for you, it must be tough."""
-11_voice_NPC_5003_104#9,"""Hey partner. So much has changed since we became ARKS hasn't it?"""
+Afin_mb14_020#0,"""Ah, it's not really a "get rich quick" thing. That gets old pretty fast. """
```

Comment:
you need to escape the quotes https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki#user-content-reminders-

Reply:
I think I corrected it.